packages/config/src/layer2s/aztec.ts

```diff
@@ -131,8 +131,8 @@ export const aztec: Layer2 = {
         },
       ],
     },
-    validatorFailure: {
-      ...RISK_VIEW.VALIDATOR_PROPOSE_BLOCKS_ZKP,
+    proposerFailure: {
+      ...RISK_VIEW.SELF_PROPOSE_ZK,
```

Comment:
Nit pick: I like how previously the name for the `validatorFailure` always started with `VALIDATOR`. It kinda gives it structure. With so many risks I think it makes sense to keep that, WYT?